src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/demo_db?useSSL=false
+spring.datasource.username=root
+spring.datasource.password=123456
+
+
+## Hibernate Properties
+# The SQL dialect makes Hibernate generate better SQL for the chosen database
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5InnoDBDialect
```

Comment:
don't place `space` before and after `=`

Reply:
Fixed